Audio/WAVFileReader.cpp

```diff
@@ -16,15 +16,9 @@
 
 using namespace DirectX;
 
-#ifndef MAKEFOURCC
```

Comment:
I had a number of places where I defined this macro, so I moved it to PlatformHelpers.h